web/src/invite.ts

```diff
@@ -411,8 +411,6 @@ function open_invite_user_modal(e: JQuery.ClickEvent<Document, undefined>): void
             set_expires_on_text();
         });
 
-        $("#expires_on").text(valid_to(Number.parseFloat($expires_in.val()!)));
```

Comment:
`set_expires_on_text` does this and is called above in the post render logic, so this seemed redundant. But maybe there's something I missed.

Reply:
Looks correct to remove.